scripts/docker-testnet/functions.sh

```diff
@@ -6,14 +6,18 @@
 IP_HOST_BYTE=3
 
 cloneRepositories() {
-  if [[ -n $CI_RUN ]]; then
-    echo "Repositories have been cloned in the CI"
-  else
-    cd $(dirname $MULTIVERSXDIR)
-
-    git clone git@github.com:multiversx/mx-chain-deploy-go.git || true
-    git clone git@github.com:multiversx/mx-chain-proxy-go.git || true
-  fi
+  cd $(dirname $MULTIVERSXDIR)
```

Comment:
Why this change? Does this function will **always** clone the proxy and deploy repos?

Reply:
yes, it will. But if the repository already exists, it won't clone them again.

Reply:
confirmed that is working properly :+1: 
```
git clone git@github.com:multiversx/mx-chain-deploy-go.git || true
fatal: destination path 'mx-chain-deploy-go' already exists and is not an empty directory.
```